packages/actors_api/src/credential/Credential.ts

```diff
@@ -11,15 +11,18 @@
  * @preferred
  */
 
```

Comment:
This could all live in `AttestedClaim`, or `AttestedClaim` in core could be renamed to `Credential` and contain the methods here.
We should also keep the concept of making a Presentation.
Let's have a follow-up ticket here.